### PR TITLE
2020 3dep proposal

### PR DESCRIPTION
**Pull request by steveoh**
I am not sure how but I would like to notify Anna about the post so she can look it over.

After today I am gone for a while so the sooner we can resolve this PR and get it merged the better.

https://deploy-preview-1250--gis-utah-gov.netlify.com/3-dep-lidar-proposal-2020/

### CODE REVIEW

File: _posts/2019-12-30-3-dep-lidar-proposal-2020.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+-	Kane County (QL2) - 6,055
+-	Strawberry River (QL1 & 4-band aerial photography) - 88
+Proposed Projects
+-	Wasatch Back (QL1 or QL2) - 1,103
+- Iron County (QL1 & QL2) - 3,067
+- NRCS (QL1 or QL2) - 2,706
+-	Sanpete & Sevier County (QL2) - 2,397
+- Garfield County (QL2) - 2,891
+
+ A shapefile of the proposed Lidar coverage areas is available for [download](ftp://ftp.agrc.utah.gov/LiDAR/3DEP_Lidar_Proposal_2020.zip).
```
Comment (steveoh) [inline _posts/2019-12-30-3-dep-lidar-proposal-2020.md]: should we put this zip file in GCP?

Comment (rkelson) [reply _posts/2019-12-30-3-dep-lidar-proposal-2020.md]: No I do not want to use GCP for single file things like this, only collections of data but I might change my mind down the road

File: _posts/2019-12-30-3-dep-lidar-proposal-2020.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  email: rkelson@utah.gov
+date: 2019-10-29 22:15:55
+categories:
+- Featured
+tags:
+- lidar
+- 3dep
+---
+
+In October a partnership of Utah organizations submitted a proposal to the [USGS 3D Elevation Program (3DEP) FY19/20 Broad Agency Announcement (BAA)](https://www.usgs.gov/core-science-systems/ngp/3dep/fy1920-usgs-broad-agency-announcement-baa-3d-elevation-program-3dep) for over 18,217 square miles for 2020 Lidar acquisition projects. A couple projects, Kane County (Greater Sevier Addition) and Strawberry River, have already begun.
```
Comment (atribolet) [inline _posts/2019-12-30-3-dep-lidar-proposal-2020.md]:
```suggestion
In October a partnership of Utah organizations submitted a proposal to the [USGS 3DEP FY19/20 BAA](https://www.usgs.gov/core-science-systems/ngp/3dep/fy1920-usgs-broad-agency-announcement-baa-3d-elevation-program-3dep) for over 18,217 square miles for 2020 Lidar acquisition projects. A couple projects (i.e., Kane County [Greater Sevier Addition] and Strawberry River) have already begun.
```

Comment (atribolet) [reply _posts/2019-12-30-3-dep-lidar-proposal-2020.md]: (Adding acronyms at end of article)

File: _posts/2019-12-30-3-dep-lidar-proposal-2020.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+While we're months away from any preliminary or final funding decisions, the partnership is sharing [this proposed project area map]({{ "/images/3DEP_Lidar_Proposal_2020.png" | prepend: site.baseurl }})
+ in advance, in an effort to promote coordination and avoid duplication of effort in these areas.
```
Comment (atribolet) [inline _posts/2019-12-30-3-dep-lidar-proposal-2020.md]:
```suggestion
 in advance in an effort to promote coordination between the partners and avoid duplication of effort in these project areas.
```

File: _posts/2019-12-30-3-dep-lidar-proposal-2020.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+The proposed areas break down to the following square mileage:
```
Comment (atribolet) [inline _posts/2019-12-30-3-dep-lidar-proposal-2020.md]:
```suggestion
The proposed areas have the following square mileage:
```

Comment (atribolet) [reply _posts/2019-12-30-3-dep-lidar-proposal-2020.md]: For some reason "break down to" was throwing me off here. Maybe this is a preferential edit, and if the original wording is preferred, that's fine; just a suggestion!

File: _posts/2019-12-30-3-dep-lidar-proposal-2020.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+[![3DEP Proposal]({{"/images/3DEP_Lidar_Proposal_2020_sm.png" | prepend:site.baseurl}} "2020 3DEP Lidar Proposal A - click for larger image")]({{"/images/3DEP_Lidar_Proposal_2020.png" | prepend:site.baseurl}}){:.inline-text-right}
+
+Projects Already Underway
+
+- Kane County (QL2) - 6,055
+- Strawberry River (QL1 & 4-band aerial photography) - 88
```
Comment (atribolet) [inline _posts/2019-12-30-3-dep-lidar-proposal-2020.md]:
```suggestion
- Strawberry River (QL1 and 4-band aerial photography) - 88
```

File: _posts/2019-12-30-3-dep-lidar-proposal-2020.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+
+Proposed Projects
+
+- Wasatch Back (QL1 or QL2) - 1,103
+- Iron County (QL1 & QL2) - 3,067
```
Comment (atribolet) [inline _posts/2019-12-30-3-dep-lidar-proposal-2020.md]:
```suggestion
- Iron County (QL1 and QL2) - 3,067
```

File: _posts/2019-12-30-3-dep-lidar-proposal-2020.md [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+- NRCS (QL1 or QL2) - 2,706
+- Sanpete & Sevier County (QL2) - 2,397
```
Comment (atribolet) [inline _posts/2019-12-30-3-dep-lidar-proposal-2020.md]: Should this be Sanpete and Sevier Counties?

**Comment by atribolet** [comment]
Forgot to add the acronyms! @steveoh Not sure of the best way to add these in. Could you add them please? 

*[BAA]: Broad Agency Announcement
*[USGS]: United States Geological Survey
*[3DEP]: 3D Elevation Program
*[QL]: Quality Level

File: _posts/2019-12-30-3-dep-lidar-proposal-2020.md (hunk 7 above)
Comment (steveoh) [reply _posts/2019-12-30-3-dep-lidar-proposal-2020.md]: probably

```suggestion
- Sanpete and Sevier Counties (QL2) - 2,397
```

**Pull request merged**